subsys/bootloader/bl_validation/bl_validation.c

```diff
@@ -113,6 +113,12 @@ static bool validate_firmware(u32_t fw_dst_address, u32_t fw_src_address,
 		return false;
 	}
 
+	if (fwinfo->valid != 0xFFFFFFFF) {
```

Comment:
Lets use some define instead of a magic value for `0xFFFFFFFF`

Reply:
Done